package.json

```diff
@@ -85,6 +85,7 @@
     "gendocs": "typedoc --options typedoc.js src/ && node scripts/post-gendocs.mjs",
     "lint:eslint": "eslint 'src'",
     "lint:prettier": "prettier -c .",
+    "prettier": "prettier --write --list-different .",
```

Comment:
missing command for prettification

Reply:
I just did `yarn lint:prettier -w`. but this works too sure